.github/workflows/full-test.yml

```diff
@@ -36,26 +33,27 @@ jobs:
           git config --global user.email "neo_ci@fake_mail.com"
           git config --global user.name "neo CI"
           # create an environement and install everything
-          python -m venv test_env
-          source ./test_env/bin/activate
+          python -m venv ~/test_env
+          source ~/test_env/bin/activate
           python -m pip install --upgrade pip
           pip install -r requirements.txt
           pip install -r requirements_testing.txt
           pip install -e .
+      - name: Get ephy_testing_data current head hash
+        # the key depend on the last comit repo https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git
+        id: vars
+        run: |
+          echo "::set-output name=HASH_EPHY_DATASET::$(git ls-remote https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git HEAD | cut -f1)"
       - uses: actions/cache@v2
         id: cache-datasets
-        env:
-          # the key depend on the last comit repo https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git
-          HASH_EPHY_DATASET: git ls-remote https://gin.g-node.org/NeuralEnsemble/ephy_testing_data.git HEAD | cut -f1
         with:
           path: ~/ephy_testing_data
-          key: ${{ runner.os }}-datasets-${{ env.HASH_EPHY_DATASET }}
+          key: ${{ runner.os }}-datasets-${{ steps.vars.outputs.HASH_EPHY_DATASET }}
           restore-keys: |
             ${{ runner.os }}-datasets
       - name: Test with pytest
         run: |
-          source ./test_env/bin/activate
-          # onbly rawiotest is tested at the moment
+          source ~/test_env/bin/activate
+          # onbly neo.rawio and neo.io
           pytest --cov=neo neo/test/rawiotest
-          # TODO uncomment this soon
-          # pytest --cov=neo neo/test/iotest
+          pytest --cov=neo neo/test/iotest
```

Comment:
I think just using `pytest` here would be sufficient since the coverage information is not used later...

Reply:
OK